charts/fybrik-crd/templates/app.fybrik.io_fybrikapplications.yaml

```diff
@@ -247,9 +247,6 @@ spec:
                   additionalProperties:
                     description: DatasetDetails holds details of the provisioned storage
                     properties:
-                      datasetRef:
```

Comment:
you changed the CRD `Status`. if there are CRD instances with this field, users will get an error.

Reply:
Restored the field.